ui/src/adapters/api/connections.ts

```diff
@@ -52,7 +55,7 @@ export async function getConnection({
       },
       method: "GET",
       signal,
-      url: `${API_VERSION}/connections/${id}`,
+      url: `${API_VERSION}/${identifier}/connections/${id}`,
```

Comment:
How can an `identifier` param used to build a URL be `null`?

I'd remove the type `Identifier` from the domain and use the `string` type everywhere.

Also, I'd rename the param `identifier` as `issuerIdentifier` everywhere. The `identifier` alone does not say much about the param.

Reply:
I did this because there is a case where the identifier is null. When the user does not select an issuer, in this case he will only see the page to select or create an issuer, and before he does so, the identifier will be null

Reply:
Ok, we need to think a bit more about this logic and how to properly model it, but we can not end up with a required value such as this identifier that can be `null` app-wide. Do nothing about this for now. I'll propose something when I fully understand the case.

Reply:
I agree with you, will change to empty string instead of null